rasa/utils/tensorflow/models.py

```diff
@@ -97,9 +96,13 @@ def __init__(self, random_seed: Optional[int] = None, **kwargs: Any) -> None:
         self._checkpoint = tf.train.Checkpoint(model=self)
 
     def _set_random_seed(self) -> None:
-        random.seed(self.random_seed)
-        tf.random.set_seed(self.random_seed)
-        np.random.seed(self.random_seed)
+        tf.keras.utils.set_random_seed(self.random_seed)
+        tf.experimental.numpy.random.seed(self.random_seed)
+        # When running on the CuDNN backend, two further options must be set
+        os.environ["TF_CUDNN_DETERMINISTIC"] = "1"
+        os.environ["TF_DETERMINISTIC_OPS"] = "1"
+        # Set a fixed value for the hash seed
+        os.environ["PYTHONHASHSEED"] = str(self.random_seed)
```

Comment:
So the `random.seed` and `np.random.seed` is not necessary?

Reply:
Right, because `tf.keras.utils.set_random_seed(self.random_seed)` inside calls something similar to:
```
import random
import numpy as np
import tensorflow as tf
random.seed(seed)
np.random.seed(seed)
tf.random.set_seed(seed)
```

Reply:
I decided to make it explicit, and I returned them.